tests/unit/test_utils.py

```diff
@@ -434,3 +434,23 @@ def test_sort(self):
         data = dict(y=2, x=1)
         assert fmf.utils.dict_to_yaml(data) == "y: 2\nx: 1\n"
         assert fmf.utils.dict_to_yaml(data, sort=True) == "x: 1\ny: 2\n"
+
+
+class TestSplitPatternRepl:
+    @pytest.mark.parametrize("src", [
+        "",  # Empty input
+        "/",  # Not long enough input
+        ";/;",  # Not long enough - missing 'REPL;'
+        ";;;",  # PATTERN and REPL parts empty
+        "/a/b/c",  # Input after trailing deliminer
+        "/a/b/c/d",  # More parts after trailig delimiter
```

Comment:
```suggestion
        "/a/b/c/d/",  # More than 3 delimiters
```

Isn't the original one basically identical to `/a/b/c`? I think you want to test this case instead?